tests/phpunit/MaxTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\AzureStorageTableExtractor\Tests;
+
+use Keboola\AzureStorageTableExtractor\IncrementalFetchingHelper;
+use PHPUnit\Framework\Assert;
+use PHPUnit\Framework\TestCase;
+
+class MaxTest extends TestCase
+{
+    /**
+     * @dataProvider getData
+     * @param mixed $value1
+     * @param mixed $value2
+     * @param mixed $expected
+     */
+    public function testMax($value1, $value2, $expected): void
+    {
+        Assert::assertSame($expected, IncrementalFetchingHelper::max($value1, $value2));
+        Assert::assertSame($expected, IncrementalFetchingHelper::max($value2, $value1));
+    }
+
+    public function getData(): iterable
```

Comment:
chtělo by to otestovat i případ kdy je max hodnota ve `value1` a kladný desetinný čísla

Reply:
vdaka, ... to uz je otestovane: https://github.com/keboola/ex-azure-storage-table/blob/f8feabeb3b67d8be82d7d9b036e68a9c048df0bd/tests/phpunit/MaxTest.php#L21-L22

desatine cisla pridam.

Reply:
jo vidíš to... koukal jsem jen na hodnoty a tohle jsem přehlídl... super 👍